src/structures/Interaction.js

```diff
@@ -197,6 +197,25 @@ class Interaction extends Base {
       MessageComponentTypes[this.componentType] === MessageComponentTypes.SELECT_MENU
     );
   }
+
+  /**
+   * Indicates whether this interaction is a {@link ModalTextInput}.
+   * @returns {boolean}
+   */
+  isTextInput() {
+    return (
+      InteractionTypes[this.type] === InteractionTypes.MESSAGE_COMPONENT &&
+      MessageComponentTypes[this.componentType] === MessageComponentTypes.INPUT_TEXT
+    );
+  }
```

Comment:
This is not added in the typings yet, and `ModalTextInput` is a component class, not an interaction class. Is there supposed to be a `TextInputInteraction`?

Reply:
No, the interaction is on submission of the whole modal, MODAL_SUBMIT

Reply:
Shouldn't this method be removed then?